standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/utils/MetaStoreUtils.java

```
@@ -291,9 +291,17 @@ public static boolean isTargetOfReplication(Database db) {
     return dbParameters != null && !StringUtils.isEmpty(dbParameters.get(ReplConst.TARGET_OF_REPLICATION));
   }
 
+  public static boolean isBackgroundThreadsEnabledForRepl(Database db) {
+    assert (db != null);
```

won't be executed in production code, should we throw an exception here?